stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3821 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        team_map_matrix = filtered_all_teams_map_data.set_index("team_name").pivot(
</pre><p>Maybe worth an extra comment to explain what pivot is doing here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Comment added